src/css/rent-form.css

```diff
@@ -1,4 +1,103 @@
-.sec-rent-form {
-  background-color: black;
-  color: wheat;
+.rent-form {
+  padding-left: 20px;
+  padding-right: 20px;
+  background-color: #242424;
+}
+.rent-title{
+  font-weight: 500;
+font-size: 28px;
+line-height: 1;
+letter-spacing: -0.02em;
+color: #fbfbfb;
+}
+.rent-formular{
+  display: flex;
+align-items: flex-start;
+justify-content: flex-start;
+flex-direction: column;
+gap: 18px;
+width: 336px;
```

Comment:
width 336px not good. rubber layout from 320px to 375px, use min-width and max width here or in the .container, it's your decision.